feat: fcc-todo-app (#51367)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo/64ec9c55fdeef78bacd2fc3b.md

---
id: 64ec9c55fdeef78bacd2fc3b
title: Step 24
challengeType: 0
dashedName: step-24
---

# --description--

> If you attempt to add another task now, you'll see the values you entered for the previous task. So, you need to clear the input fields after adding a task.

Reviewer: ```suggestion
If you attempt to add another task now, you'll notice that the input fields retain the values you entered for the previous task. To resolve this, you need to clear the input fields after adding a task.
```

> Instead of clearing the input fields one by one, you can create a function where you clear those fields, and then call that function whenever you want to clear the fields again.

Reviewer: ```suggestion
Instead of clearing the input fields one by one, it's a good practice to create a function that handles clearing those fields. You can then call this function whenever you need to clear the input fields again.
```
